src/actix/api/service_api.rs

```diff
@@ -87,10 +88,18 @@ async fn get_locks(toc: web::Data<TableOfContent>) -> impl Responder {
     process_response(Ok(result), timing)
 }
 
+#[get("/stacktrace")]
```

Comment:
Should we list this in our OpenAPI specification? [Here](https://github.com/qdrant/qdrant/blob/master/openapi/openapi-service.ytt.yaml) for example.

Reply:
That's a debug-only API. For internal use, not for users